packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -442,6 +442,22 @@ class Auth {
         }
     }
 
+    logTokenRequestError = async (error: Error) => {
+        // the regular error.message will return only the generic status code message
+        // ie. 'Bad Request' for 400. We need to drill specifically into the ResponseError
+        // to get a more descriptive error message from SLAS
+        if (error instanceof Error && 'response' in error) {
+            // commerce-sdk-isomorphic throws a `ResponseError`, but doesn't export the class.
```

Comment:
We control this package so there's nothing stopping from exporting it :D 

but the release wouldn't make it in time for this PR, so maybe we can export it in the next release and follow up in another PR if we really want to